tests/modules/collaborations/test_models.py

```diff
@@ -27,13 +27,13 @@ def test_collaboration_create_with_members(
     request.addfinalizer(basic_collab.delete)
 
     assert len(basic_collab.get_users()) == 2
+    assert basic_collab.initiator_guid == collab_user_a.guid
```

Comment:
Anyone any idea 
1 how did this work? The association lost the initiator field a long time ago
2 how did the changes above get it to start working properly

Reply:
What do you mean?  I still see the `initiator_guid` field in the `Collaboration` model

Reply:
There is NOW :) 
That was added a while ago. Previously there was an initiator Boolean field on the association which the tset below was  validating but that has been removed a while agoo.... and this test still passed

Reply:
Oh I guess your comment was meant for line 33:

```python
assert association.initiator is True
```

It's because of this line https://github.com/WildMeOrg/houston/blob/16c97b958c419c1416e6856ed65f1af848e3d044/app/modules/collaborations/models.py#L112

Reply:
Thanks, that was definitely a bug. Will fix that.